subjects/setspace/README.md

```diff
@@ -0,0 +1,55 @@
+## setspace
+
+### Instructions
+
+Write a function that takes a PascalCase `string` and returns the same `string` with a space between each word.
+
+- The function must return an empty `string` if the input `string` is empty.
+
+- The function must return `"Error"` if the input `string` is not a PascalCase `string`.
+
+- The PascalCase begins with a capital letter, and each word begins with a capital letter without a space between them for example: `"HelloWorld`"` is a valid PascalCase `string`.
```

Comment:
Please fix this line and the next one, and make sure you run prettier afterwards.